curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/finding-a-remainder-in-javascript.md

```diff
@@ -13,12 +13,20 @@ The <dfn>remainder</dfn> operator `%` gives the remainder of the division of two
 
 **Example**
 
-<blockquote>5 % 2 = 1 because<br>Math.floor(5 / 2) = 2 (Quotient)<br>2 * 2 = 4<br>5 - 4 = 1 (Remainder)</blockquote>
+```js
+const five = 5;
+const two = 2;
+const quotient = Math.floor(5 / 2);
+console.log(quotient); // logs 2
+```
 
 **Usage**  
 In mathematics, a number can be checked to be even or odd by checking the remainder of the division of the number by `2`.
 
-<blockquote>17 % 2 = 1 (17 is Odd)<br>48 % 2 = 0 (48 is Even)</blockquote>
+```js
+const oddNumber = 17;
+const evenNumber = 48;
```

Comment:
Shouldn't example operations with `% 2` be included here?

Reply:
I've used what was suggested in the issue description. Can something like this work better?
```
const oddNumber = 17 % 2;
console.log(oddNumber); // logs 1 (17 is Odd)
const evenNumber = 48 % 2;
console.log(evenNumber); // logs 0 (48 is Odd)
```

Reply:
To be honest trying to write it as javascript will only make it harder to read. It's just couple math equations.

Reply:
This is meant to fix the RTL layout as mentioned in the issue description. I'm a first-timer and don't know what to do at this point. Should I close the pull and go back to discuss the issue?
Also, thanks for the help!